src/reference/changes.rst

```diff
@@ -38,6 +38,18 @@ Cylc 8.2.0
    :cylc-rose: `1.3 <https://github.com/cylc/cylc-rose/blob/1.3.x/CHANGES.md>`__
 
 
+Upgrade To The Latest Jupyter Releases
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Cylc UI Server has been updated to work with the latest releases of
+`Jupyter Server`_ and `Jupyter Hub`_.
+
+If you are utilising Cylc's multi-user functionality then your configuration
+will require some changes to work with these releases.
+
+See :ref:`cylc.uiserver.multi-user` for more details
+
+
```

Comment:
Shouldn't this be under 8.3 rather than 8.2?

Reply:
Yes, it got delayed.